Client crashed when you click on the tray icon before it is fully loaded

The client doesn't like it when you press the system tray icon before the client is loaded.

![error](https://user-images.githubusercontent.com/36420562/37929420-3ea2c6f2-3140-11e8-893e-3422c03ad30a.PNG)
```
java.lang.ArrayIndexOutOfBoundsException: 102510
	at ab.x(ab.java:1118)
	at client.ah(client.java:920)
	at bh.h(bh.java:369)
	at bh.copy$run(bh.java:323)
	at bh.run(bh.java:60)
	at java.lang.Thread.run(Thread.java:748)
```

And then this happens, the client does not start
![error_2](https://user-images.githubusercontent.com/36420562/37929484-61e0f0a8-3140-11e8-932a-b8c86086ee1a.PNG)

```
Exception in thread "main" java.lang.reflect.InvocationTargetException
	at java.awt.EventQueue.invokeAndWait(EventQueue.java:1349)
	at java.awt.EventQueue.invokeAndWait(EventQueue.java:1324)
	at javax.swing.SwingUtilities.invokeAndWait(SwingUtilities.java:1348)
	at net.runelite.client.ui.ClientUI.show(ClientUI.java:344)
	at net.runelite.client.RuneLite.start(RuneLite.java:187)
	at net.runelite.client.RuneLite.main(RuneLite.java:122)
Caused by: java.awt.IllegalComponentStateException: The frame is displayable.
	at java.awt.Frame.setUndecorated(Frame.java:923)
	at net.runelite.client.ui.ClientUI.lambda$show$8(ClientUI.java:346)
	at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:301)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:758)
	at java.awt.EventQueue.access$500(EventQueue.java:97)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.awt.EventQueue$3.run(EventQueue.java:703)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:80)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:728)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:205)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
```
Reproduce:
1. Start the client
2. Wait for the tray icon to appear and click on it

OS: Windows 10
Java: 8_162

--- a/runelite-client/src/main/java/net/runelite/client/ui/ClientUI.java
+++ b/runelite-client/src/main/java/net/runelite/client/ui/ClientUI.java
@@ -301,8 +301,6 @@ public class ClientUI
 			// Try to enable fullscreen on OSX
 			OSXUtil.tryEnableFullscreen(frame);
 
-			trayIcon = SwingUtil.createTrayIcon(ICON, properties.getTitle(), frame);
-
 			frame.setTitle(properties.getTitle());
 			frame.setIconImage(ICON);
 			frame.getLayeredPane().setCursor(Cursor.getDefaultCursor()); // Prevent substance from using a resize cursor for pointing
@@ -399,6 +397,8 @@ public class ClientUI
 			frame.toFront();
 			requestFocus();
 			giveClientFocus();
+
+			trayIcon = SwingUtil.createTrayIcon(ICON, properties.getTitle(), frame);
 		});
 
 		eventBus.post(new ClientUILoaded());

